### DISCUSSION TITLE
Problem with react-lottie/lottie-react package

### DISCUSSION THREAD

**Discussion by shahriarsajeeb** [vercel/next.js#65205] [App Router]
<img width="1413" alt="image" src="https://github.com/vercel/next.js/assets/87035691/9efd23bb-f794-440f-a154-7c6db09134f5">

When I integrate the react-lottie package with the latest version of Next.js, it throws an error during the project build. However, after implementing dynamic import, the error disappears in production mode, but amusingly, it resurfaces in development mode! 😂 The development mode error reference is mentioned above!

**Comment by ali9821** [comment]
Use lottie-react instead of react-lottie
Also import it with "Next/dynamic" to avoid Vercel refrenceError: document is not defined

**Reply by rakib13x** [reply]
still it gives error
